articles/configuring-maintenance-windows-in-fleet.md

```diff
@@ -0,0 +1,111 @@
+
+Detailed instructions for configuring your Google Calendar account, along with pre-formatted API keys, are provided within Fleet.
+
+To access these steps, click the top right **User logo** > **Settings** page > **Integrations** tab > **Calendars** on the left-hand menu.
+
+Follow the instructions found on this page.
+
+> Please note that the Fleet server will use APIs provided by Google Calendar. Some additional load should be expected on the server. Make sure to have enough quota. (This can be adjusted later per need)
+
+Video: [How to configure the Google Calendar API in Fleet](https://www.youtube.com/watch?v=CpbzJNrM2LY). (2m)
+
+### Creating a webhook
+
+An external webhook mechanism should be defined to be run as the mitigation plan for failing policies on hosts.
+
+Please refer to the [Building webhook flows with Fleet and Tines](https://fleetdm.com/guides/building-webhook-flows-with-fleet-and-tines) guide to learn more about building webhooks using Tines.
+
+At the end of this operation, you should have a webhook to be used by the Fleet server.
+
+### Configuring a Policy to run the Webhook if failing (Admin)
+
+1. On your Fleet UI, go to the **Policies** tab.
+2. On the top left dropdown, select a team if needed.
+3. On the top right, click **Manage automations** → **Calendar events**.
+4. On the top left corner of the newly opened modal, click **Enable**.
+5. Paste the webhook you have in the box named: **Resolution webhook URL**
+6. Select one or more desired policies (click the checkbox next to it)
+
+Video: [Setting a webhook for failing policies in Fleet](https://www.youtube.com/watch?v=evMB6zC8jso). (3m)
+
+### Which users will have an event set for them?
+
+If a policy on a host belonging to a Google Calendar user fails, an event will be set on their calendar next Tuesday between 9 a.m. and 5 p.m. local time.
+
+If more than one host has the same user, the event will only be set for one of the hosts. Once this host has been remedied, another event will be scheduled for the next of the user’s hosts, and so forth.
+
+If more than one user is set for a specific host, the event will be set for one of the users.
+
+### What details will be presented in the calendar event?
+
+When adding a policy (**Policies** tab > Top right corner **Add policy** > **create the policy** > Press **Save**), a modal will open with the policy details like this:
+
+![Save policy dialog with Antivirus healthy for Linux with example AI-generated description and resolution.](../website/assets/images/articles/configuring-maintenance-windows-in-fleet-651x782@2x.png "Save policy dialog")
+
+The description and resolution fields will be AI-generated based on the SQL on the policy.
+
+The admin may press **Autofill** again to regenerate or edit the text manually. Learn more about [Fleet’s AI-assisted policy descriptions and resolutions](https://fleetdm.com/guides/fleet-ai-assisted-policy-descriptions-and-resolutions).
```

Comment:
You can't press **Autofill** twice AFAIK

Reply:
Ahhh, maybe we discussed this as a future thingy ?

Reply:
@spokanemac, this leaves only the manual edit. Sorry.

Reply:
Confirmed, you can only press once; once saved, it is only editable.

This has been updated to read: "The admin may press **Autofill** one time to regenerate or edit the text manually."